src/plugins/performanceIndicator/plugin.js

```diff
@@ -58,5 +65,58 @@ export default function PerformanceIndicator() {
         indicator.statusClass('s-status-error');
       }
     }
+
+    function showOverlay() {
+      const overlayStylesText = `
+          #c-performance-indicator--overlay {
+            background-color:rgba(0,0,0,0.5);
+            position: absolute;
+            width: 300px;
+            left: calc(50% - 300px);
+          }
+      `;
+      const overlayMarkup = `
+        <div id="c-performance-indicator--overlay" aria-label="Performance Overlay">
+          <table id="c-performance-indicator--table">
+            <tr class="c-performance-indicator--row"><td class="c-performance-indicator--measurement-name"></td><td class="c-performance-indicator--measurement-value"></td></tr>
+          </table>
+        </div>
+      `;
+      const overlayTemplate = document.createElement('div');
+      overlayTemplate.innerHTML = overlayMarkup;
+      const overlay = overlayTemplate.cloneNode(true);
+      overlay.querySelector('.c-performance-indicator--row').remove();
+      const overlayStyles = document.createElement('style');
+      overlayStyles.appendChild(document.createTextNode(overlayStylesText));
+
+      document.head.appendChild(overlayStyles);
+      document.body.appendChild(overlay);
+
+      indicator.off('click', showOverlay);
+
+      const interval = setInterval(() => {
+        overlay.querySelector('#c-performance-indicator--table').innerHTML = '';
+
+        for (const [name, value] of openmct.performance.measurements.entries()) {
+          const newRow = overlayTemplate
+            .querySelector('.c-performance-indicator--row')
+            .cloneNode(true);
+          newRow.querySelector('.c-performance-indicator--measurement-name').innerText = name;
+          newRow.querySelector('.c-performance-indicator--measurement-value').innerText = value;
+          overlay.querySelector('#c-performance-indicator--table').appendChild(newRow);
+        }
+      }, 1000);
```

Comment:
this is the configured interval?

Reply:
No, this is just a rendering loop, it has nothing to do with the batching interval. 

This performance overlay is a crude debugging interface to allow us to tune batching post-deployment.

Reply:
I will make it a const though, because no magic numbers.